### DISCUSSION TITLE
vs2022

### DISCUSSION THREAD

**Discussion by zj951555793** [microsoft/vcpkg#31513] [General]
I am currently using vs2022, but what should I do if I want to use the 2019 toolset? CMake uses V142

**Comment by zj951555793** [comment]
When I used the 2022 library to compile another library in 2019, I was able to successfully compile and produce the exe correctly, but it would crash during runtime.

I think it's caused by mixing libraries. My computer is equipped with VS2022 and usually uses the VS2019 toolset to compile programs

**Comment by autoantwort** [comment]
You can set https://learn.microsoft.com/en-us/vcpkg/users/triplets#vcpkg_platform_toolset in an own triplet.
